packages/mobile/src/components/audio/GoogleCast.tsx

```diff
@@ -80,7 +78,7 @@ export const useChromecast = () => {
         })
       }
     },
-    [client, streamingUri, owner]
+    [client, owner]
```

Comment:
just checking we dont want contentUrl to be in effect?

Reply:
it's an arg to the fn, so nope!